packages/core/upload/admin/src/pages/App/components/Filters.tsx

```diff
@@ -38,11 +42,11 @@ export const Filters = () => {
       <FilterPopover
         displayedFilters={displayedFilters}
         filters={filters}
+        onToggle={setOpen as FilterPopoverProps['onToggle']}
         onSubmit={handleSubmit}
-        onToggle={setOpen}
       />
       <FilterList
-        appliedFilters={filters}
+        appliedFilters={filters as FilterListProps['appliedFilters']}
```

Comment:
I think this is the second time we have to type cast these props. I don't think it makes sense that every time we want to use this component we have to type case the prop we are passing.

Reply:
see this comment

https://github.com/strapi/strapi/pull/21896#discussion_r1827523924